README.md

```diff
@@ -6,47 +6,57 @@ Contents:
-- `cliff vesting`: When the `owner` create a `cliff vesting`, it'll be allowed to add a series of point which contains time and amount. Then the vesting will calculate the claimable tokens by these points.
-- `linear vesting`: When the `owner` create a `linear vesting`, it'll be allowed to set start time and end time. Then the vesting will calculate the claimable tokens by start time and end time.
+The `owner` can create as many vesting as he/she wants. A vesting includes a beneficiary and necessary settings based on its type. The vesting types are as the following:
 
-### Freeze a vesting
+- `cliff vesting`: This type of vesting allows the owner to add a series of `release point` which contains release time and amount. Then the vesting will calculate the claimable tokens by these `release point`s.
+- `linear vesting`: This type of vesting allows the owner to set start time and end time. Then the vesting will calculate the claimable tokens by start time and end time linearly.
 
-- The `owner` can freeze a vesting, then the beneficiary will not be allowed to claim the tokens when the vesting is frozen.
+### Pause and resume a vesting
+
+- The `owner` can pause a vesting, then the beneficiary can not claim the tokens from the vesting anymore until the vesting is resumed.
+- A paused vesting can be resumed by the `owner`.
 
 ### Terminate a vesting
 
-- The `owner` can terminate a vesting, then the vesting will be deleted and the unclaimed tokens will belong to the `owner`.
+- The `owner` can terminate a vesting, then the beneficiary can not claim the tokens from the vesting anymore.
+- This is an one-time action. A termiated vesting can not be activated again.
 
 ### Claim tokens
 
-- The `beneficiary` can claim tokens from a vesting. The vesting will calculate the claimable token and then send tokens to the beneficiary's account.
+- A `beneficiary` of a vesting in this contract can claim tokens from a vesting. The vesting will calculate the claimable token and then send tokens to the beneficiary's account.
 
+> Is it possible to support that the beneficiary can claim tokens in all vestings which are with the same beneficiary?
 
-### Change beneficiary
+### Change beneficiary of a vesting
 
 - The `beneficiary` and the `owner` can set a new `beneficiary` in a vesting. Then the next time claimed tokens will send to the new `beneficiary`.
 
+### Withdraw remaining tokens in the contract
+
+The owner can withdraw the remaining tokens in this contract only if there is no active/paused vesting in this contract.
 
 ### View functions
 
-This contract has a set of view functions for anyone to get the status detail of this contract.
\ No newline at end of file
+This contract has a set of view functions for anyone to get the status detail of this contract.
+
+> Maybe should add a view function for beneficiary to query the claimable amount of a vesting or the total claimable amount in all vestings.
```

Comment:
This function might be better implemented on the front end? Because the front end has calculated the claimable amount of single vesting. So it just needs to summarize in the front end.